### PR TITLE
Addition to note

### PR DESCRIPTION
**Pull request by joinimran**
As the policy needs to have Windows 10 ver 1803 or later. Or The ADMX template file needs to be updated. 

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/9726

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/windows-defender-security-center/wdsc-device-security.md [hunk 1]
```diff
@@ -86,6 +82,8 @@ If you don't want users to see the recommendation to update TPM firmware, you ca
 
 ## Disable Memory integrity switch
 If you don't want users to be able to change the Hypervisor Control Integrity (HVCI), or memory integrity, setting on their computers, you can disable the **Memory integrity** switch.
+> [!IMPORTANT]
+> You must have Windows 10, version 1803 or later. The ADMX/ADML template files for earlier versions of Windows do not include these Group Policy settings.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-security-center/wdsc-device-security.md:86]:
```suggestion
> You must have Windows 10, version 1803 or later. The ADMX/ADML template files for earlier versions of Windows do not include these Group Policy settings. 
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by denisebmsft — APPROVED** [review]
Thank you @joinimran @JohanFreelancer9 @e0i.

**Comment by denisebmsft** [comment]
@tiburd I am going to go ahead and merge this pull request.

**Pull request merged**